modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/actions.js

```diff
@@ -16,5 +16,5 @@ export const CREATE_SEGMENTS_EXPERIENCE = 'CREATE_SEGMENTS_EXPERIENCE';
 export const SELECT_SEGMENTS_EXPERIENCE = 'SELECT_SEGMENTS_EXPERIENCE';
 export const DELETE_SEGMENTS_EXPERIENCE = 'DELETE_SEGMENTS_EXPERIENCE';
 export const UPDATE_SEGMENTS_EXPERIENCE = 'UPDATE_SEGMENTS_EXPERIENCE';
-export const UPDATE_SEGMENTS_EXPERIENCE_PRIORITY =
-	'UPDATE_SEGMENTS_EXPERIENCE_PRIORITY';
+export const UPDATE_SEGMENTS_EXPERIENCES_LIST =
+	'UPDATE_SEGMENTS_EXPERIENCES_LIST';
```

Comment:
Is there any reason to have two similar files? Maybe we can get rid of one :)

Reply:
It's true! I didn't realize that! 😱 😂 I'll remove one. Thanks!